app/routes/tasks_routes.py

```diff
@@ -0,0 +1,134 @@
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, abort, make_response, request
+from datetime import *
+import requests
+import os
+
+
+tasks_bp = Blueprint("tasks_bp",__name__, url_prefix="/tasks")
+
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message":f"{cls.__name__} {model_id} invalid"}, 400))
+
+    model = cls.query.get(model_id)
+
+    if not model:
+        abort(make_response({"message":f"{cls.__name__} {model_id} not found"}, 404))
+
+    return model
+    
+
+@tasks_bp.route("",methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+
+    try:
+        new_task = Task.from_dict(request_body)
+    except:
+        return {"details": "Invalid data"} , 400
+    
+    db.session.add(new_task)
+    db.session.commit()
```

Comment:
Great job using `db.session.<method-name>.` Session gives us access to the following:
* `db` is our app’s instance of SQLAlchemy.
* `session` represents our active database connection.
* By referencing db.session we can use SQLAlchemy’s methods to perform tasks like committing a change to a model, storing a new record of a model, and deleting a record.

By referencing `db.session.add()` you are able to use the SQLAlchemy method to store a new record of the Task model. ⭐